spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/support/converter/SimplePubSubMessageConverterTests.java

```diff
@@ -106,7 +106,7 @@ public void testNullHeaders() {
 		SimplePubSubMessageConverter converter = new SimplePubSubMessageConverter();
 
 		// test extraction from PubsubMessage to T
-		assertEquals(TEST_STRING, toString.convert(converter.fromPubSubMessage(PubsubMessage.newBuilder()
+		assertEquals(TEST_STRING, toString.convert((T) converter.fromPubSubMessage(PubsubMessage.newBuilder()
```

Comment:
I think we need to reflect reality in the test as well - and get rid of the generics here too!

Reply:
This would cause me to do a bunch more casts in the individual tests. Is there sufficient benefit?